cdap-ui/app/cdap/components/PipelineDetails/RunLevelInfo/RunComputeProfile/index.js

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright © 2018 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+*/
+
+import React, {Component} from 'react';
+import {connect} from 'react-redux';
+import PropTypes from 'prop-types';
+import {objectQuery} from 'services/helpers';
+import {PROFILE_NAME_PREFERENCE_PROPERTY} from 'components/PipelineConfigurations/ConfigurationsContent/ComputeTabContent/ProfilesListView.js';
```

Comment:
nit: don't need the `.js`. Actually, since it seems like we'll be using this constant quite a lot, why don't we move it to somewhere more generic like `PipelineConfigurations/Store/ActionCreator`, instead of having it in this very specific component?

Reply:
👍 Yes. This because of vscode auto import. My bad. However I think the constant should be in that particular module as it is used only by the profiles in pipeline and is not used anywhere in the `PipelineConfigurations/Store/`.